tests/test-sources/plugins/pluginmanagers/lazy.nix

```diff
@@ -90,10 +98,73 @@
     };
   };
 
+  name-only-plugin = {
+    plugins.lazy = with pkgs.vimPlugins; {
+      enable = true;
+      plugins = [
+        {
+          name = "echasnovski/mini.ai";
+          pkg = mini-nvim;
+          enabled = false;
+        }
+        {
+          name = "echasnovski/mini.ai";
+          enabled = false;
+        }
+      ];
+
+    };
+  };
+
+  dir-only-plugin = {
+    plugins.lazy = with pkgs.vimPlugins; {
+      enable = true;
+      plugins = [ { dir = "${LazyVim}"; } ];
+    };
+  };
+
```

Comment:
Such a test-case would be inherently impure, and therefore potentially flakey, so we could have one to test the module eval, but we should set `test.runNvim = false` to avoid actually running nvim and downloading the URL.

Since (in tests) nvim runs in a derivation builder, I don't think we'd be able to download anyway because I'm pretty sure buildscripts are sandboxed from the internet to enforce purity.